src/src/assets/scss/mixins.scss

```diff
@@ -0,0 +1,237 @@
+@import './variables.scss';
+@import './functions.scss';
+
+/// Mixin to prefix a property
+/// @author Hugo Giraudel
+/// @param {String} $property - Property name
+/// @param {*} $value - Property value
+/// usage @include vendor-prefix('box-sizing', 'border-box');
+@mixin prefix($property, $value, $important: false) {
+    @each $prefix in ('webkit', 'moz', 'ms', 'o') {
+        #{'-' + $prefix + '-' + $property}: $value;
+    }
+
+    // Output standard non-prefixed declaration
+    @if $important {
+        #{$property}: $value !important;
+    }
+
+    @else {
+        #{$property}: $value;
+    }
+}
+
+@mixin flexbox {
```

Comment:
Using prefixes for flexbox is not necessary anymore. Browsers these days support flexbox. Unless this app is used on older browsers, (IE10 and lower). Then you need something special for that.